src/framework/components/collision/component.js

```diff
@@ -123,6 +126,13 @@ class CollisionComponent extends Component {
         }
     }
 
+    onSetOffset(name, oldValue, newValue) {
+        this._isOffset = true;
```

Comment:
```suggestion
        this._isOffset = !this.data._linearOffset.equals(pc.Vec3.ZERO) || !this.data._angularOffset.equals(pc.Quat.IDENTITY);
```

Reply:
I obviously haven't tested the suggestion above - but something like that handles the case where any offset is removed.

Reply:
Hey @LeXXik - what do you think about my comments on this line. If the offsets are reset to zero, _isOffset needs to be set to false.

Reply:
Probably the component data? I'm not sure. The `CollisionComponent` is one of the few remaining components that still uses the `ComponentData` block.

Reply:
I think we should check the `newValue` on zero equality, which is a Vec3 for both properties.

Reply:
I will need to create a test to verify this.